libraries/src/AWS.Lambda.Powertools.Parameters/AWS.Lambda.Powertools.Parameters.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="AWSSDK.AppConfigData" Version="3.7.0.110" />
+      <PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.5.10" />
```

Comment:
Can you update to the latest versions of these packages.
AWSSDK.DynamoDBv2 -> 3.7.103.1
AWSSDK.SecretsManager -> 3.7.102.38
AWSSDK.SimpleSystemsManagement -> 3.7.104.29